src/main/java/ui/screens/HomePage.java

```diff
@@ -0,0 +1,88 @@
+package ui.screens;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+
+public class HomePage extends BasePage {
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    public WebElement buttonsButton() {
+        return getElement("(//li[@id='item-4'])[1]");
+    }
+
+    public WebElement clickMeButton() {
+        return getElement("(//*[contains(text(),'Click Me')])[3]");
+    }
+
+    public WebElement clickMessageText() {
+        return getElement("//p[@id='dynamicClickMessage']");
+    }
+
+    public WebElement addButton() {
+        return getElement("//button[@id='addNewRecordButton']");
+    }
+
+    public WebElement registrationFormTitle() {
+        return getElement("//div[@id='registration-form-modal']");
+    }
+
+    public WebElement firstNameField() {
+        return getElement("//input[@id='firstName']");
+    }
+
+    public WebElement lastNameField() {
+        return getElement("//input[@id='lastName']");
+    }
+
+    public WebElement emailField() {
+        return getElement("//input[@id='userEmail']");
+    }
+
+    public WebElement ageField() {
+        return getElement("//input[@id='age']");
+    }
+
+    public WebElement salaryField() {
+        return getElement("//input[@id='salary']");
+    }
+
+    public WebElement departmentField() {
+        return getElement("//input[@id='department']");
+    }
+
+    public WebElement submitButton() {
+        return getElement("//button[@id='submit']");
+    }
+
+    public WebElement firstNameInTable() {
+        return getElement("//div[contains(@class,'rt-tr -even')]/div[contains(text(),'Jhon')]");
```

Comment:
Please, do not use searching by text in locators, avoid this for fields which you need to check